<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Gabor噪声控制台</string>
  </property>
  <widget class="QLabel" name="kLabel">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>0</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gabor核
Gabor Kernel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kpLabel">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>0</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>核的频谱
Spectrum</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>256</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gabor噪音
Gabor Noise</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="npLabel">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>256</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>噪音的频谱
 Spectrum of Noise</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>256</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
      <item>
       <widget class="QToolButton" name="kbtn">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>K</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ksb">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <item>
       <widget class="QToolButton" name="abtn">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="asb">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
      <item>
       <widget class="QToolButton" name="fbtn">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>F0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fsb">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.250000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
      <item>
       <widget class="QToolButton" name="wbtn">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ω0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="wsb">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
      <item>
       <widget class="QToolButton" name="krl">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>kernel_radius</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="krsb">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>2.000000000000000</double>
        </property>
        <property name="maximum">
         <double>64.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>4.000000000000000</double>
        </property>
        <property name="value">
         <double>8.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
      <item>
       <widget class="QToolButton" name="npcl">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>number_of_impulses_per_cell</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="npcsb">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>2048.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,2,2">
      <item>
       <widget class="QCheckBox" name="icb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>各向同性</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="texBox">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>黑白噪声</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>噪声上色</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>花瓶条纹</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>云朵纹理</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>火焰纹理</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="genBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>生成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="saveBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
